### DISCUSSION TITLE
Allow stability days to be configurable on a per-package level

### DISCUSSION THREAD

**Discussion by Jacob-Roberts** [renovatebot/renovate#14351] [Request Help] [answered]
### What would you like Renovate to be able to do?

I like having my stability days set to 3 days, however for some internal packages that my team publishes we want it to merge immediately (after CI). Currently there is no way to set the stabilityDays property inside the packageRules objects. 
I would love to be able to have a configuration like this:
```json
{
  "packageRules": [
    {
      "matchPackagenames": ["my-internal-package"],
      "stabilityDays": 0
    }
  ],
  "stabilityDays": 3
}
```

### If you have any ideas on how this should be implemented, please tell us here.

I'm not sure. I'm just a user of renovate currently

### Is this a feature you are interested in implementing yourself?

Maybe

**Comment by viceice** [comment, marked as answer]
`stabilityDays` is supported in package rules. Why are you thinking it's not?

**Reply by Jacob-Roberts** [reply]
Oh maybe the json schema doesn’t have it listed, my intellisense made me feel like it was not supported

**Reply by Jacob-Roberts** [reply]
If this is correct, can we update the jsonSchema to have stabilityDays listed?

**Reply by viceice** [reply]
Sure, but schema generation is very tricky. So that's probably not easy to solve.

**Reply by rarkins** [reply]
You're welcome to take a look though, it's all in this repo

**Reply by Jacob-Roberts** [reply]
Am I correct in assuming that "schedule" is also supported on a packageRules level scope?

**Reply by Jacob-Roberts** [reply]
Never mind, found that in the documentation. It does work on a packageRules scope

**Reply by TWiStErRob** [reply]
Note for future readers: `stabilityDays` has been [renamed](https://github.com/renovatebot/renovate/pull/21376#issuecomment-1505778643) to [minimumReleaseAge](https://docs.renovatebot.com/configuration-options/#minimumreleaseage) in [Renovate 35.46.0](https://github.com/renovatebot/renovate/releases/tag/35.46.0)
